src/Flysystem/LocalFilesystemAdapter.php

```diff
@@ -27,4 +47,75 @@ public function prefixPath(string $path): string
     {
         return $this->pathPrefixer->prefixPath($path);
     }
+
+    /**
+     * List the contents of a path by glob.
+     *
+     * @return iterable<StorageAttributes>
+     */
+    public function listContentsByGlob(string $folder, string $fileGlob, bool $deep = FilesystemReader::LIST_SHALLOW): iterable
+    {
+        $basePath = $this->prefixPath($folder);
+        if (!is_dir($basePath)) {
+            return [];
+        }
+
+        // Use the glob to match the base path first.
+        yield from $this->doTheThing(Path::join($basePath, $fileGlob));
```

Comment:
Rename method

Reply:
You already said that in https://github.com/silverstripe/silverstripe-assets/pull/685#discussion_r2094754045 :p